### PR TITLE
Update and rename content/packages/working-with-a-github-packages-reg…

### PR DESCRIPTION
**Pull request by e52x**
…istry/working-with-the-rubygems-registry.md to isi/paket/bekerja-dengan-github-paket-registry/working-with-the-rubygems-registry.md

Vvvv

### Why:

Closes [issue link]

### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Review by louanne2022 — COMMENTED** [review]
Hi I'm trying to install types of keys . I have difficulty installing y keys.

**Review by Barnes617 — COMMENTED** [review]
Approved

File: isi/paket/bekerja-dengan-github-paket-registry/working-with-the-rubygems-registry.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ shortTitle: RubyGems registry
 ---
 
 {% data reusables.package_registry.packages-ghes-release-stage %}
-{% data reusables.package_registry.packages-ghae-release-stage %}
```
Comment (Barnes617) [inline isi/paket/bekerja-dengan-github-paket-registry/working-with-the-rubygems-registry.md:20]: >   @@iso/packet/procrastinating/working-with-the-rubygems-registry.md

**Review by Barnes617 — COMMENTED** [review]
[-with-the-rubygems-registry.md](barnesbuilders5.webnode.com) 
#16542 
> Link to 
[googlecloud_unified_analytics_data_platform_paper_2021.pdf](https://github.com/github/docs/files/8355400/googlecloud_unified_analytics_data_platform_paper_2021.pdf)

**Review by louanne2022 — COMMENTED** [review]
I would like to know if there's something I can pay on GitHub .Two have my certificate and key codes installed on my account with GitHub .You can contact me by email .My email is lou_annedure@hotmail.com .

**Pull request closed without merging**